client/js/components/week/week-board.js

```diff
@@ -0,0 +1,21 @@
+/** @jsx React.DOM */
+var React = require('react');
+var WeekDayCard = require('./week-day-card');
+var _ = require('../../utils/utils');
+
+var WeekBoard = React.createClass({
+  render: function(){
+    var weekDayCards = _.map(this.props.data, function(item, key){
+      return <WeekDayCard key={ key } data={ item } />;
```

Comment:
in place of 'data', i'd recommend using a more descriptive prop name - I think it makes the react code more readable later on (in week-day-card.js, the code also maps over this.props.data, it'd be cool if it was more clear exactly what we are iterating over )